client/src/components/scores/scores.js

```diff
@@ -3,23 +3,37 @@
 export default {
 	template: `
 	<div class="top-bar secondary">
-	    <div class="top-bar-left">
-	        <input name="search" type="text" ng-model="scores.search" placeholder="search Scores" />
-	        <button class="button" ng-hide="scores.duplicateScores().length === 0" ng-click="scores.showDuplicates = !scores.showDuplicates">
-	            {{ scores.showDuplicates ? 'Show all scores' : 'Show only duplicates' }}
-	        </button>
-	        <button class="button" ng-hide="scores.errorScores().length === 0"
-	        ng-click="scores.showErrors = !scores.showErrors">
-	        {{ scores.showErrors ? 'Show all scores' : 'Show only errors'}}
-	        </button>
-	    </div>
-	</div>
-	<div class="top-bar-page">
-	    <div id="scores-list" class="grid-x grid-padding-x small-up-1 medium-up-3 large-up-5">
-	        <div ng-repeat="score in scores.scores()" class="cell grid-y">
-	            <score id="score-{{score._id}}" data="score" class="card"></score>
-	        </div>
-	    </div>
-	</div>`,
+    <div class="top-bar-left">
+        <ul class="menu">
+            <li>
+                <input name="search" type="text" ng-model="scores.search" placeholder="search Scores" />
+            </li>
+            <li>
+                <select ng-disabled="!scores.teamIsSelected()" ng-model="scores.round">

```

Comment:
The entity which this PR is about, is defined in the rules for the past few years as a `match`. That's also its name in the server. Please change all naming to be the same - Match.